pydis_site/apps/api/serializers.py

```diff
@@ -293,7 +293,7 @@ def update(self, instance: QuerySet, validated_data: list) -> list:
                 try:
                     user = instance_mapping[user_data["id"]]
                 except KeyError:
-                    raise NotFound({"id": f"User with id {user_data['id']} not found."})
+                    raise NotFound({"detail": [f"User with id {user_data['id']} not found."]})
```

Comment:
Using a list is a `ValidationError` convention only
```suggestion
                    raise NotFound({"detail": f"User with id {user_data['id']} not found."})
```